hazelcast/src/main/java/com/hazelcast/concurrent/countdownlatch/operations/CountDownLatchBackupOperation.java

```diff
@@ -20,13 +20,15 @@
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.spi.BackupOperation;
+import com.hazelcast.spi.impl.MutatingOperation;
 
 import java.io.IOException;
 
 import static com.hazelcast.concurrent.countdownlatch.CountDownLatchDataSerializerHook.COUNT_DOWN_LATCH_BACKUP_OPERATION;
 import static java.lang.Boolean.TRUE;
 
-public class CountDownLatchBackupOperation extends AbstractCountDownLatchOperation implements BackupOperation {
+public class CountDownLatchBackupOperation extends AbstractCountDownLatchOperation
```

Comment:
do we perform quorum checks for backup operations?

Reply:
yeah, that was the policy before. We discussed it with @jerrinot but he said not to change how we handle it currently. I was thinking of doing a revision of all operations afterwards. But don't know if @jerrinot wants to change it still in 3.10